### PR TITLE
MAP-1605 Add Prison Register integration and refactor docker setup

### PR DESCRIPTION
**Pull request by Mjwillis**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,17 @@ LOCATIONS_INSIDE_PRISON_API_CLIENT_SECRET=
...
-docker compose -f docker-compose-local.yml up
+docker compose up
 ```
 
 Then run the API.
 
+### Running the whole setup in docker
+
+```shell
+docker compose --profile include-api build
```
Comment (jimbali) [inline README.md:37]: Won't `docker compose build` just build it but not run it?

Comment (Mjwillis) [reply README.md:37]: yes this is true! oops

File: README.md [hunk 2]
```diff
@@ -26,11 +26,17 @@ LOCATIONS_INSIDE_PRISON_API_CLIENT_SECRET=
...
-docker compose -f docker-compose-local.yml up
+docker compose up
 ```
 
 Then run the API.
 
+### Running the whole setup in docker
+
+```shell
+docker compose --profile include-api up
```
Comment (jimbali) [inline README.md:37]: You can also do
```
docker compose --profile include-api up --build
```
to make sure it does a fresh build when running it, or even
```
docker compose --profile include-api up --build --push
```
to keep the deps up-to-date

**Review by jimbali — APPROVED** [review]
Approved, with comment

File: README.md (hunk 2 above)
Comment (Mjwillis) [reply README.md:37]: I'll add to next PR!

**Pull request merged**